sysl2/sysl/tests/integration_with_restrict_by.sysl

```diff
@@ -0,0 +1,19 @@
+IntegratedSystem [appfmt="project", epfmt="project ep", r="y"]:
```

Comment:
```
appfmt="%(appname)"
epfmt="%(epname)"
```

Reply:
appfmt and epfmt should only be on the Project endpoint.

Reply:
This test case does not show how restrict_by works.

Reply:
changed